Poker.BE/Poker.BE.Service/Modules/Requests/LogoutRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Poker.BE.Service.Modules.Requests
+{
+	public class LogoutRequest : IRequest
+	{
+		public int User { get; set; }
```

Comment:
👎 please add `///<summery>` to explain to people this is a hashcode of a user. not actual user yet